plugins/dayspan.config.js

```diff
@@ -54,9 +54,25 @@ export default {
       };
     },
 
-    getEventOccurrence()
+    getEventOccurrence(schedule, start, labels, formats)
     {    
-      return "";
+      const multiplicator = 1.125;
+      const shiftingHours = 6;
```

Comment:
Ähnlicher Code ist jetzt in 3 Variationen hier, im `store/splus.ts` und in `getStyleNow()` 🙅‍♂️ 
Kannst du das als Methoden zueinander in eine neue Datei packen oder so 😬

Reply:
🙅‍♂️